tracecat/actions/integrations/edr/sentinel_one.py

```diff
@@ -49,6 +50,21 @@
 """
 
 
+def create_sentinel_one_client() -> httpx.AsyncClient:
+    SENTINEL_ONE_API_TOKEN = os.getenv("SENTINEL_ONE_API_TOKEN")
+    if SENTINEL_ONE_API_TOKEN is None:
+        raise ValueError("SENTINEL_ONE_API_TOKEN is not set")
+    client = httpx.AsyncClient(
+        base_url=f'{os.getenv("SENTINEL_ONE_BASE_URL")}/web/api/v2.1',
```

Comment:
@mattdurant are all S1 API endpoints under v2.1?

Reply:
All that I'm aware of, v2.0 was missing a lot of the newer feature endpoints and there was some refactoring as well.  I think there are a lot of integrations out there for other security tools still using v2.0 and haven't been upgraded to use 2.1.

Reply:
In that case, shall we parameterize `api_version` in `create_sentinel_one_client(api_version: str ="v2.1")` keep v2.1 as default to allow this to be reuse for any non-updated APIs?

Reply:
It wouldn't hurt, I'm not sure I would ever use it, but I can follow that pattern when working on integrations